pkg/services/object/put/slice.go

```diff
@@ -21,23 +21,23 @@ type slicingTarget struct {
 	maxObjSize       uint64
 	homoHashDisabled bool
 
-	nextTarget transformer.ObjectTarget
+	nextTarget internal.Target
 
 	payloadWriter *slicer.PayloadWriter
 }
 
-// returns transformer.ObjectTarget for raw root object streamed by the client
+// returns [objectcore.Target] for raw root object streamed by the client
```

Comment:
Is it `internal` or `objectcore`?

Reply:
has not pushed the last commit rebase, fixed